src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1246,6 +1252,10 @@ def _configure_security_policy_storage_params(arg_ctx):
                    arg_type=get_enum_type(ServerConnectionType),
                    help='The connection type used for connecting to the instance.')
 
+        c.argument('minimal_tls_version',
+                   arg_type=get_enum_type(SqlManagedInstanceMinimalTlsVersionType),
+                   help='The minimal TLS version enforced by the managed instance for inbound connections.')
```

Comment:
You already defined this in `sql server` scope, why do you need to repeat this in `sql server create` scope?

Reply:
I think this is the `sql mi` scope, please correct me if I'm wrong